### PR TITLE
Update mitiq bounties 2

### PR DESCRIPTION
**Pull request by nathanshammah**
We are updating the bountied issues on Mitiq, to avoid any ambiguity that may have arisen in the previous ones.

### CODE REVIEW

File: participating-projects.md [hunk 1]
```diff
@@ -61,15 +61,15 @@ Current quantum computers are noisy due to interactions with the environment, im
...
+
+- [Add type check to CI](https://github.com/unitaryfund/mitiq/issues/489)
 
 #### $25 each
 
-- [Add type check to CI](https://github.com/unitaryfund/mitiq/issues/489)
 - [Add an XACC executor example and item to software list](https://github.com/unitaryfund/mitiq/issues/357)
-- [Testing or updating or replacing the TensorFlow Quantum executor example](https://github.com/unitaryfund/mitiq/issues/559)
 - [Warn users when their programs are too short](https://github.com/unitaryfund/mitiq/issues/275)
+- [Finish the implementation of Braket conversions](https://github.com/unitaryfund/mitiq/issues/275)
```
Comment (nathanshammah) [inline participating-projects.md]:
```suggestion
- [Finish the TODOs in the Braket conversions PR](https://github.com/unitaryfund/mitiq/pull/590)
```

**Review by crazy4pi314 — APPROVED** [review]
LGTM, Thanks for capturing this from the meeting 👏

File: participating-projects.md [hunk 2]
```diff
@@ -61,15 +61,15 @@ Current quantum computers are noisy due to interactions with the environment, im
...
+
+- [Add type check to CI](https://github.com/unitaryfund/mitiq/issues/489)
 
 #### $25 each
 
-- [Add type check to CI](https://github.com/unitaryfund/mitiq/issues/489)
 - [Add an XACC executor example and item to software list](https://github.com/unitaryfund/mitiq/issues/357)
-- [Testing or updating or replacing the TensorFlow Quantum executor example](https://github.com/unitaryfund/mitiq/issues/559)
 - [Warn users when their programs are too short](https://github.com/unitaryfund/mitiq/issues/275)
+- [Finish the TODOs in the Braket conversions PR](https://github.com/unitaryfund/mitiq/pull/590)
```
Comment (rmlarose) [inline participating-projects.md:72]: You beat me @crazy4pi314 :) there is now an issue for this. https://github.com/unitaryfund/mitiq/issues/679

**Comment by crazy4pi314** [comment]
Hah sorry 🤦‍♀️ merging now

**Comment by crazy4pi314** [comment]
actually @rmlarose this is closed, I cant merge the change from here. maybe make a new one quick?

**Pull request merged**